contrib/experimental/great_expectations_experimental/expectations/expect_query_count_with_filter_to_meet_threshold.py

```diff
@@ -75,7 +75,12 @@ def _validate(
 
     examples = [
         {
-            "data": {"col1": [1, 1, 1, 2, 2, 2, 2, 2]},
+            "data": [
+                {
+                    "dataset_name": "test",
+                    "data": {"col1": [1, 1, 1, 2, 2, 2, 2, 2]},
+                },
+            ],
```

Comment:
Can you add `"suppress_test_for": ["bigquery"],` before the `"tests": [` line, then delete the two only_for lines

```
--- a/contrib/experimental/great_expectations_experimental/expectations/expect_query_count_with_filter_to_meet_threshold.py
+++ b/contrib/experimental/great_expectations_experimental/expectations/expect_query_count_with_filter_to_meet_threshold.py
@@ -24,7 +24,7 @@ class ExpectQueryCountWithFilterToMeetThreshold(QueryExpectation):
     metric_dependencies = ("query.template_values",)

     query = """
-                SELECT COUNT(1)
+                SELECT COUNT(*) n
                 FROM {active_batch}
                 WHERE {col} = {filter}
             """
@@ -77,10 +77,10 @@ class ExpectQueryCountWithFilterToMeetThreshold(QueryExpectation):
         {
             "data": [
                 {
-                    "dataset_name": "test",
                     "data": {"col1": [1, 1, 1, 2, 2, 2, 2, 2]},
                 },
             ],
+            "suppress_test_for": ["bigquery"],
             "tests": [
                 {
                     "title": "basic_positive_test",
@@ -91,7 +91,6 @@ class ExpectQueryCountWithFilterToMeetThreshold(QueryExpectation):
                         "threshold": 4,
                     },
                     "out": {"success": True},
-                    "only_for": ["sqlite", "spark"],
                 },
                 {
                     "title": "basic_negative_test",
@@ -102,7 +101,6 @@ class ExpectQueryCountWithFilterToMeetThreshold(QueryExpectation):
                         "threshold": 4,
                     },
                     "out": {"success": False},
-                    "only_for": ["sqlite", "spark"],
                 },
             ],
         },
```